Author: Cyril Brulebois <kibi@debian.org>
Description: Fixing FTBFS on kfreebsd (Closes: #585969).

diff -Naurp lxtask.orig/src/main.c lxtask/src/main.c
--- lxtask.orig/src/main.c	2012-04-15 09:07:42.237134013 +0000
+++ lxtask/src/main.c	2012-04-15 09:20:15.854195792 +0000
@@ -56,7 +56,7 @@ guint win_height;
 guint refresh_interval;
 guint rID;
 
-int PAGE_SIZE;
+int page_size;
 
 int main (int argc, char *argv[])
 {
@@ -72,7 +72,7 @@ int main (int argc, char *argv[])
 #endif
     gtk_init (&argc, &argv);
 
-    PAGE_SIZE=sysconf(_SC_PAGESIZE)>>10;
+    page_size=sysconf(_SC_PAGESIZE)>>10;
     own_uid = getuid();
 
     config_file = g_build_filename(g_get_user_config_dir(), "lxtask.conf", NULL);
diff -Naurp lxtask.orig/src/types.h lxtask/src/types.h
--- lxtask.orig/src/types.h	2012-04-15 09:07:42.237134013 +0000
+++ lxtask/src/types.h	2012-04-15 09:20:15.846195723 +0000
@@ -79,6 +79,6 @@ extern guint win_height;
 const gchar *custom_signal_0;
 const gchar *custom_signal_1;
 
-extern int PAGE_SIZE;
+extern int page_size;
 
 #endif
diff -Naurp lxtask.orig/src/xfce-taskmanager-linux.c lxtask/src/xfce-taskmanager-linux.c
--- lxtask.orig/src/xfce-taskmanager-linux.c	2012-04-15 09:07:42.237134013 +0000
+++ lxtask/src/xfce-taskmanager-linux.c	2012-04-15 09:20:15.846195723 +0000
@@ -47,8 +47,8 @@ void get_task_details(gint pid,struct ta
         sscanf(line,"%d %d",&task->size,&task->rss);
         close(fd);
         if(!task->size) return;
-	task->size*=PAGE_SIZE;
-	task->rss*=PAGE_SIZE;
+	task->size*=page_size;
+	task->rss*=page_size;
 
 	sprintf(line,"/proc/%d/stat",pid);
 	fd=open(line,O_RDONLY);
