Origin: Cyril Brulebois
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=585969
Reviewed-By: Andrew Lee
Last-Update: 2010-07-31

--- a/src/types.h
+++ b/src/types.h
@@ -79,6 +79,6 @@
 const gchar *custom_signal_0;
 const gchar *custom_signal_1;
 
-extern int PAGE_SIZE;
+extern int page_size;
 
 #endif
--- a/src/xfce-taskmanager-linux.c
+++ b/src/xfce-taskmanager-linux.c
@@ -47,8 +47,8 @@
         sscanf(line,"%d %d",&task->size,&task->rss);
         close(fd);
         if(!task->size) return;
-	task->size*=PAGE_SIZE;
-	task->rss*=PAGE_SIZE;
+	task->size*=page_size;
+	task->rss*=page_size;
 
 	sprintf(line,"/proc/%d/stat",pid);
 	fd=open(line,O_RDONLY);
--- a/src/main.c
+++ b/src/main.c
@@ -56,7 +56,7 @@
 guint refresh_interval;
 guint rID;
 
-int PAGE_SIZE;
+int page_size;
 
 int main (int argc, char *argv[])
 {
@@ -72,7 +72,7 @@
 #endif
     gtk_init (&argc, &argv);
 
-    PAGE_SIZE=sysconf(_SC_PAGESIZE)>>10;
+    page_size=sysconf(_SC_PAGESIZE)>>10;
     own_uid = getuid();
 
     config_file = g_build_filename(g_get_user_config_dir(), "lxtask.conf", NULL);
